packages/adapter-knex/README.md

```diff
@@ -37,46 +15,61 @@ const keystone = new Keystone({
   name: 'My Awesome Project',
   adapter: new KnexAdapter(),
 });
-
-const uri = 'postgres://keystone5:k3yst0n3@127.0.0.1:5432/ks5_dev';
-const client = 'postgres';
-const schemaName = 'keystone';
-
-const knexOptions = { ... };
-
-keystone.connect(uri, {
-  client,
-  schemaName,
-  ...knexOptions,
-});
 ```
 
-## API
+## API `new KnexAdapter(options)`
 
-### `uri`
+### `schemaName`
```

Comment:
Where did `schemaName` come from? It's not clear where to set this variable. I find it helps to tie it back to the api like so:

```suggestion
### `options.schemaName`
```